stream-chat-android/src/main/java/com/getstream/sdk/chat/ChatImpl.java

```diff
@@ -148,7 +148,7 @@ public ChatMarkdown getMarkdown() {
     @NotNull
     @Override
     public String getVersion() {
-        return BuildConfig.BUILD_TYPE + ":" + BuildConfig.VERSION_NAME;
+        return BuildConfig.STREAM_CHAT_UI_VERSION + "-" + BuildConfig.BUILD_TYPE;
```

Comment:
Updated the version string format to match the format in the other two libraries